doc/_scripts/software_maturity/software_maturity_features.yaml

```
@@ -104,6 +104,10 @@ features:
         - SHIELD_NRF7002EB
         - BOARD_NRF7002DK_NRF5340_CPUAPP
         - BOARD_NRF7002DK_NRF5340_CPUAPP_NRF7001
+    Thread Co-existence:
```

Can you also change line 99 as **Bluetooth LE coexistence** ?
```suggestion
    Thread coexistence:
```